pages/dkp/kommander/1.4/tutorials/login-cluster/index.md

```diff
@@ -58,11 +58,38 @@ $ kubectl get pods -A
 Error from server (Forbidden): pods is forbidden: User "user@yourcompany.com" cannot list resource "pods" in API group "" at the cluster scope
 ```
 
+### Downloading a kubeconfig from the UI
+
+You can use the UI to download certain managed clusters' kubeconfigs.
+
+If you are in the **Clusters** page from the **Global** workspace view, or the **Clusters** page from any workspace with clusters, you can click on the three button action menu from the card, and select **Download kubeconfig**.
+
+   ![Kommander Cards View](/dkp/kommander/1.4/img/download-kubeconfig-clusters-page.png)
+
+If you are in the detailed page for the specific cluster, you can click on the top right **Actions** dropdown menu, and select **Download kubeconfig**.
+
+  ![Kommander cluster details View](/dkp/kommander/1.4/img/download-kubeconfig-details-page.png)
+
+After it is downloaded, you can apply this kubeconfig to your `kubectl`.
+
+If you select the **Download kubeconfig** from the UI and Kommander presents a modal that says the kubeconfig is not available for download, you must retrieve the Kubernetes credentials using the [Access a cluster instructions][access-cluster].
```

Comment:
Do we have an explanation as to why this doesn't always work?  It might be nice to lead with that, if it's not too much 'because reasons'. :-)

Reply:
Let's open a JIRA for this.